### PR TITLE
Add max height to comments

### LINKED ISSUE #567: Add max-height to comments
If some user creates a very long comment, the scroll gets really painful. We should set a max-height to comments and a Read more at the end to show the entire comment.

largecomment

### PR DESCRIPTION
**Pull request by camus-code**
Resolves #567

### CODE REVIEW

**Comment by dajohi** [comment]
@camus-code rebase needed.

File: src/components/snew/ThingComment.js [hunk 1]
```diff
@@ -15,6 +15,24 @@ class ThingComment extends React.PureComponent {
     e && e.preventDefault && e.preventDefault();
     this.props.onCensorComment(this.props.loggedInAsEmail, this.props.token, this.props.id);
   }
+  componentDidMount(){
```
Comment (fernandoabolafio) [inline src/components/snew/ThingComment.js]: Could you extract and encapsulate this logic into another function? 
something like: 
```js
handleCommentMaxHeight = () => {
//logic goes here
}
componentDidMount() {
  this.handleCommentMaxHeight()
}
```

File: src/components/snew/ThingComment.js [hunk 2]
```diff
@@ -15,6 +15,24 @@ class ThingComment extends React.PureComponent {
...
+    const insertAfter = (newNode, referenceNode) => referenceNode.parentNode.insertBefore(newNode, referenceNode.nextSibling);
+    const commentsDiv = Array.prototype.slice.call(document.querySelectorAll(".comment > .entry > .usertext"));
+    const overflowDivs = commentsDiv.filter(function isOverflown(element) {
+      return element.scrollHeight > element.clientHeight || element.scrollWidth > element.clientWidth;
+    });
+    for (let i = 0; i < overflowDivs.length; i++) {
```
Comment (fernandoabolafio) [inline src/components/snew/ThingComment.js]: A suggestion: here you could use the `forEach` method instead, for easier readability:
```js
overflowDivs.forEach((overflowDiv, index) => {
// logic goes here
})
```

**Comment by camus-code** [comment]
Everything should be up to par now

**Pull request merged**